examples/union-type/tests/union_type_test.bal

```diff
@@ -17,14 +17,7 @@ public function mockPrint(any|error... s) {
 function testFunc() {
     // Invoking the main function
     main();
-    test:assertEquals(<string> outputs[0], "This is a string");
-    test:assertEquals(<int> outputs[1], 101);
-    test:assertEquals(<string> outputs[2], "this is a value");
-
-    var output3 = outputs[3];
-    if (output3 is error) {
-        test:assertEquals(output3.message(), "key '' not found");
-    } else {
-        test:assertFail();
-    }
+    test:assertEquals(outputs[0].toString(), "name=Peter address=50 Bridgeton Lane Tuckerton, NJ 08087");
```

Comment:
Instead of checking `toString()` shall we create two Person records and assert by value? For the error we can validate the message maybe?

Reply:
fixed.